pkg/keys/printer_test.go

```diff
@@ -57,7 +57,6 @@ func TestPrettyPrint(t *testing.T) {
 		{RangeLeaseKey(roachpb.RangeID(1000001)), "/Local/RangeID/1000001/r/RangeLease"},
 		{RangeStatsKey(roachpb.RangeID(1000001)), "/Local/RangeID/1000001/r/RangeStats"},
 		{RangeTxnSpanGCThresholdKey(roachpb.RangeID(1000001)), `/Local/RangeID/1000001/r/RangeTxnSpanGCThreshold`},
-		{RangeFrozenStatusKey(roachpb.RangeID(1000001)), "/Local/RangeID/1000001/r/RangeFrozenStatus"},
```

Comment:
Yep, and we still have it in `keys/printer.go`. I kept the decoding logic (including in `cli/debug.go`) but dropped the test since keeping the test would require us to keep a function to construct these keys (that otherwise wouldn't be used)